packages/integration-karma/test/rendering/disconnecting-root-vm/index.spec.js

```diff
@@ -0,0 +1,20 @@
+import { createElement } from 'lwc';
+import ParentCmp from 'x/parent';
+
+describe('disconnecting root vm', () => {
```

Comment:
@caridy @jodarove with the recent change, are eliminating an ability to handle a real faulty scenario when assertion is legit? For example when a node is removed programmatically and then the system is attempting to remove it, shouldn't that still cause an error? Asking because if we no longer can hit a real case removal assertion, then perhaps we should just remove the check all together; Alternatively, if there is a real scenario however, we should then have a test for it to ensure it still works.

Reply:
@apapko no, we left the assertion for vm.state == created.